sdk/java/src/main/java/org/symbol/sdk/utils/Writer.java

```diff
@@ -0,0 +1,19 @@
+package org.symbol.sdk.utils;
+
+/**
+ * This interface allows writing to an underlining storage.
+ */
+public interface Writer {
+
+	/**
+	 * Writes array into buffer.
+	 *
+	 * @param buffer Data to write.
+	 */
+	void write(final byte[] buffer);
+
+	/**
+	 * Get the underlining storage.
+	 */
+	byte[] getStorage();
```

Comment:
why do you need this on interface?

Reply:
Let me check.  I think I added it for tests, but let me check

Reply:
This will also be needed for the model class, which serializes as in the [JS SDK](https://github.com/symbol/symbol/blob/445b82ede955b0185597e3f8f890529d13b791d7/sdk/javascript/src/nem/models.js#L786), since the buffer is allocated in the Writer class.